app/views/cip_change_alert/start.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Your training and support programme has changed
+    </h1>
+    <p>
+      Your school has changed your training and support programme from <%= original_cip_name %> to <%= new_cip_name %>
+    </p>
+
+    <p>
+      You can no longer access the materials you were previously using
+    </p>
+
+    <p>
+      Speak to your mentor for more information
+    </p>
```

Comment:
Do we want dots at the end of sentences here?

Reply:
I thought that but it's not on the design.

Reply:
tell a lie, yes it is. My bad